public/pages/ReviewAndCreate/containers/ReviewAndCreate.tsx

```diff
@@ -92,28 +91,45 @@ export function ReviewAndCreate(props: ReviewAndCreateProps) {
   // meaning validation has passed and succesful callout will display or validation has failed
   // and callouts displaying what the issue is will be displayed instead.
   useEffect(() => {
-    dispatch(validateDetector(formikToDetector(props.values)))
+    dispatch(validateDetector(formikToDetector(props.values), 'model'))
       .then((resp: any) => {
         if (isEmpty(Object.keys(resp.response))) {
           setValidDetectorSettings(true);
           setValidModelConfigurations(true);
         } else {
-          if (resp.response.hasOwnProperty('detector')) {
-            const issueType = Object.keys(resp.response.detector)[0];
-            if (resp.response.detector[issueType].hasOwnProperty('message')) {
+          if (
+            resp.response.hasOwnProperty('detector') ||
+            resp.response.hasOwnProperty('model')
+          ) {
+            const validationType = Object.keys(resp.response)[0];
+            const issueType = Object.keys(resp.response[validationType])[0];
+            if (
+              resp.response[validationType][issueType].hasOwnProperty('message')
+            ) {
               const validationMessage =
-                resp.response.detector[issueType].message;
+                resp.response[validationType][issueType].message;
               const detectorSettingIssue: ValidationSettingResponse = {
                 issueType: issueType,
                 message: validationMessage,
+                validationType: validationType,
               };
+
+              // Potentially remove warning toast in these cases
+              if (issueType == 'aggregation' || issueType == 'timeout') {
```

Comment:
Why would we want to remove the warning toast here? What does an `aggregation` issue type mean?

Reply:
The reason for this is due to the fact that if there is an `aggregation` issue then it might be an issue that arouse from a change in core and user won't have much they can do about it and for timeout they also can't really re-try since action happens automatically when entering `ReviewAndCreate` page so I thought that I could just make it basically invisible to users instead of additionally giving a warning toast that will confuse them on there next action.

Reply:
In this same `if` block, I see it sets the variables indicating valid detector settings and model configs, when in fact that might not be true if there was a problem performing the queries, or if it timed out. It seems like a bad user experience to see it showing as valid in the callouts, then failing later when actually creating the detector, or trying to start it.

How about having a separate callout text indicating that validation failed to run? It could be generic so that it covers both aggregation and timeout failures. I think showing the toast with more details on the failure (aggregation or timeout) could be helpful to the user, and would allow for quick user feedback in case there is something wrong with the AD code that we would need to fix (e.g., aggregation failure where the queries being made in validation API call need to be updated after core changes something internal).

Reply:
As discussed separately, these issue types only come up during non-blocker validation, after blocker validation has passed. So in that case it is mostly validated, just not fully validated / not able to provide any recommendations in case there is data sparsity due to any of the configs / features.

I think the 2 options are (1) hide these failures like you're doing now, or (2) find a way to tell the user that blocking validation passed, but non-blocking validation failed to run properly. For simplicity, I think its ok to go with option 1 for this release, and sync with UX team on a solution to option 2 later on.

Suggest to update the comments on line 117 to reflect something along the lines of this